src/main/charts/bitbucket/values.yaml

```diff
@@ -8,14 +8,15 @@
 # - volumes
 #
 # Additional details on pre-provisioning these required resources can be found here:
-# https://atlassian.github.io/data-center-helm-charts/installation/INSTALLATION/#3-configure-database
-# https://atlassian.github.io/data-center-helm-charts/installation/INSTALLATION/#5-configure-persistent-storage
+# https://atlassian.github.io/data-center-helm-charts/userguide/PREREQUISITES/#provision-a-database
```

Comment:
@tan-ro I think I misunderstood the `pre-provisioning` here. I was thinking pre-provisioning db/volumes themselves, instead of pre-configure those resources in the product. 
So what you mean here is how to configure those stanza, right? Then links to installation steps make more sense. I'll update them.

Reply:
Great. Let me know if you need a hand